etc/setup/README.pypi.md

```diff
@@ -0,0 +1,227 @@
+# SMARTS
```

Comment:
Why is `README.pypi.md` file present in two locations, namely, `SMARTS/etc/setup/` and `SMARTS/` ?